Drop after devel/libfmt >= 9 update

CMake Error at CMakeLists.txt:205 (find_package):
  Could not find a configuration file for package "fmt" that is compatible
  with requested version "9".

  The following configuration files were considered but not accepted:

    /usr/local/lib/cmake/fmt/fmt-config.cmake, version: 8.1.1

--- CMakeLists.txt.orig	2022-12-16 19:05:00 UTC
+++ CMakeLists.txt
@@ -202,5 +202,5 @@ find_package(enet 1.3)
 # =======================================================================
 
 find_package(enet 1.3)
-find_package(fmt 9 REQUIRED)
+find_package(fmt 8 REQUIRED)
 find_package(inih)
